fmf.spec

```diff
@@ -34,6 +34,7 @@ BuildRequires: python%{python3_pkgversion}-jsonschema
 BuildRequires: git-core
 %{?python_provide:%python_provide python%{python3_pkgversion}-%{name}}
 Requires:       git-core
+Recommends:     python%{python3_pkgversion}-referencing
```

Comment:
I don't think we need it. The dependency is part of `jsonschema` as early as 4.18 [^1] and for rhel case we can just assume it does not have the interface.

[^1]: https://github.com/python-jsonschema/jsonschema/commit/69f3899c0770472f19898d80aea2b7a51fd5ec6b

Reply:
oh cool, thanks. Removing.